docs/src/app/components/pages/components/DatePicker/Page.jsx

```diff
@@ -59,6 +62,13 @@ const DatePickerPage = () => (
       <DatePickerExampleControlled/>
     </CodeExample>
     <CodeExample
+      title="Disabled dates example"
+      description={descriptions.disabledDates}
+      code={datePickerExampleDisableDatesCode}
+    >
+      <DatePickerExampleDisableDates/>
```

Comment:
As far as I know, there is no rule for this yet. We would have to submit a PR on the `eslint-plugin-react` repository.